full-service/src/bin/main.rs

```diff
@@ -109,6 +109,7 @@ fn main() {
             network_state,
             config.get_fog_pubkey_resolver(logger.clone()).map(Arc::new),
             config.num_workers,
+            config.offline,
```

Comment:
What is the use case for offline mode? I don't really understand what that is for.